package.json

```diff
@@ -186,9 +185,10 @@
     "webpack-dev-server": "^4.1.1",
     "webpack-manifest-plugin": "2.0.4"
   },
-  "resolutions": {
+  "overrides": {
     "glob-parent": "^5.1.2",
-    "react": "16.14.0",
```

Comment:
why remove existing resolutions react? isn't it backwards incompatible?

Reply:
"Overrides" is just needed to fix security vulnerabilities, As long as this is passing CI, there should not be issue.

`react` was added internally due to issue with SharedFrontendComponents dependency.